src/environments/environment.ts

```diff
@@ -5,6 +5,6 @@
 export const environment = {
   production: false,
   profileUrl:
-    'https://landville-backend-web-api.herokuapp.com/api/v1/auth/profile/',
-  api_url: 'https://landville-backend-web-api.herokuapp.com/api/v1',
+    'http://127.0.0.1:8000/api/v1/auth/profile/',
```

Comment:
Hey @shemaeric, since we are not done developing the feature we can change back to the live URLs in the environments.